src/components/TeacherPage.js

```diff
@@ -1,18 +1,93 @@
+import { useState } from "react";
+import { Settings, Setting } from "./Settings";
 import "../styles/TeacherPage.css";
 
-const TeacherPage = () => {
+const SideBar = ({ token, onLinkClick, darkMode, setDarkMode, snowFlakes, setSnowFlakes }) => {
+    return (
+        <div className="sidebar">
+            {
+                ["Dashboard", "Homework", "Students"].map((link) => {
+                    return (
+                        <div className="link-container" onClick={ () => { onLinkClick(link.toLowerCase()) } }>
+                            <h1 className="link">{ link }</h1>
+                        </div>
+                    )
+                })
+            }
+
+            <Settings token={ token } style={{bottom: '3.5%', left: '1%'}} rotation="top">
```

Comment:
Avoid using inline styles.

Reply:
Then how should i use them?

Reply:
Add this to a css file and add `className` here